frontend/src/containers/Collective.js

```diff
@@ -0,0 +1,273 @@
+import React, { Component, PropTypes } from 'react';
```

Comment:
Honestly, that doesn't really matter in this PR. This is setting up structure and adding some components that carried over. Once it's merged, we can modify individual components.